### ISSUE TITLE
Incorrect Project.toml in dev folder prevents Pkg usage

### ISSUE THREAD

**Issue by musm** [JuliaLang/Pkg.jl#660]
I had an issue today where one of the packages I was working on had an incorrectly specified Project.toml file .

(1) This effectively prevented use of the pkg manager, with error:
```julia
 Resolving package versions...
ERROR: Pkg.TOML.ParserError(186, 187, "expected `=`, but found `\n`")
```
(2) and, I had no indication as to which file  was causing the error

My suggestion
1. Better error message 
2. Perhaps allow Pkg to continue to work, skipping this project package until it is fixed with a warning message.

**Comment by KristofferC** [comment]
1) For sure
2) I think this is hard to do

**Comment by 00vareladavid** [comment]
Fixed by #925

**Comment by 00vareladavid** [comment]
Example:
```
(foobar) pkg> dev /tmp/dummy/
ERROR: Could not parse project /tmp/dummy/Project.toml: CompositeException(Any[ParserError(296, 297, "expected `=`, but found `\n`")])
```
